### PR TITLE
[Node] Add curated list to node documentation

### PR DESCRIPTION
**Pull request by emanuelcasco**
## Summary

- Add Node.js resources curated list to documentation.

### CODE REVIEW

File: nodejs/docs/curated-list.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Node.js awesome list 😎
+
+Awesome Node.js resources list curated by Wolox.
+
+## Books
+
```
Comment (gonziesc) [inline nodejs/docs/curated-list.md]: Nodejs design patterns second edition

File: nodejs/docs/curated-list.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- [Mocking ES and CommonJS modules with jest.mock()](https://medium.com/codeclan/mocking-es-and-commonjs-modules-with-jest-mock-37bbb552da43).
+- [Understanding Jest Mocks](https://medium.com/@rickhanlonii/understanding-jest-mocks-f0046c68e53c)
+- [Jest cheatsheet](https://github.com/sapegin/jest-cheat-sheet)
+
+## Talks
+
+### General
+
+- [Daniel Khan - Everything I thought I knew about the event loop was wrong](https://www.youtube.com/watch?v=gl9qHml-mKc)
```
Comment (gonziesc) [inline nodejs/docs/curated-list.md]: move this to talks maybe?

File: nodejs/docs/curated-list.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- [Error Handling](https://www.joyent.com/node-js/production/design/errors)
+
+### Testing
+
+- [Mocking ES and CommonJS modules with jest.mock()](https://medium.com/codeclan/mocking-es-and-commonjs-modules-with-jest-mock-37bbb552da43).
+- [Understanding Jest Mocks](https://medium.com/@rickhanlonii/understanding-jest-mocks-f0046c68e53c)
+- [Jest cheatsheet](https://github.com/sapegin/jest-cheat-sheet)
+
+## Talks
+
```
Comment (gonziesc) [inline nodejs/docs/curated-list.md]: https://www.youtube.com/watch?v=8aGhZQkoFbQ

Comment (emanuelcasco) [reply nodejs/docs/curated-list.md]: Added

File: nodejs/docs/curated-list.md (hunk 2 above)
Comment (emanuelcasco) [reply nodejs/docs/curated-list.md]: It's already in Talks 🍺

File: nodejs/docs/curated-list.md (hunk 1 above)
Comment (emanuelcasco) [reply nodejs/docs/curated-list.md]: Done

**Comment by gussiciliano** [comment]
Please follow the PR naming convention described in #90 :)

**Pull request closed without merging**